kie-server-parent/kie-server-router/kie-server-router-proxy/src/test/java/org/kie/server/router/repository/FileRepositoryTest.java

```diff
@@ -213,4 +214,38 @@ public void onConfigurationReloaded() {
         repoWithWatcher.close();
         repoWithWatcher.clean();
     }
+
+    @Test
+    public void testFileCreation() throws Exception {
+
+        File repositoryDirectory = new File("target" + File.separator + UUID.randomUUID().toString());
+        Configuration configuration = new Configuration();
+        ConfigurationMarshaller marshaller = new ConfigurationMarshaller();
+        File configFile = new File(repositoryDirectory, "kie-server-router.json");
```

Comment:
the idea is to test if the file is created if missing.
maybe pass to the constructor only the path without creating the file here?

Reply:
Okay, I will try this approach.